cse-lib/network-processing/src/main/java/com/farao_community/farao/cse/network_processing/ucte_pst_change/UctePstProcessor.java

```diff
@@ -29,11 +29,14 @@ public UctePstProcessor(String pstId, String nodeId) {
         this.nodeId = nodeId;
     }
 
-    public void forcePhaseTapChangerInActivePowerRegulationForIdcc(Network network) {
+    public void forcePhaseTapChangerInActivePowerRegulationForIdcc(Network network, double defaultRegulationValue) {
         TwoWindingsTransformer transformer = network.getTwoWindingsTransformer(pstId);
         PhaseTapChanger phaseTapChanger = getPhaseTapChanger(transformer);
         // PowSyBl transformer is inverted compared to UCTE transformer so we have to set opposite sign
         double regulationValue = -phaseTapChanger.getRegulationValue();
+        if (Double.isNaN(phaseTapChanger.getRegulationValue())) {
+            regulationValue = defaultRegulationValue;
+        }
```

Comment:
Maybe put the sign inversion in an else branch

Reply:
I have nothing against the idea

Reply:
got it done